docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -98,20 +103,20 @@ Fields in char1: 1
 
 ## More Advanced Types
 
-4 more advanced types are:
+Four advanced types are:
 
 - [CircuitString](../snarkyjs-reference/classes/CircuitString)
 - [PrivateKey](../snarkyjs-reference/classes/PrivateKey)
 - [PublicKey](../snarkyjs-reference/classes/Types.PublicKey)
 - [Signature](../snarkyjs-reference/classes/Signature)
 
-All arguments passed into smart contracts need to be arguments SnarkyJS can understand. This means that we can't just pass normal strings - we need to pass in strings that have been wrapped to be compatible with circuits, which is what `Struct` accomplishes.
+All arguments passed into smart contracts must be arguments SnarkyJS can consume. You cannot pass normal strings. Instead, you must pass in strings that are wrapped to be compatible with circuits. This is accomplished with `Struct`.
```

Comment:
> You cannot pass normal strings

I would rephrase this to 

> You cannot pass normal JavaScript-native types, such as strings, number, boolean or bigint

so this is more general and doesn't just focus on strings

Reply:
same here

>strings that are wrapped to be compatible with circuits